docs/source/dev_guide/release.rst

```diff
@@ -3,176 +3,83 @@ How to Prepare a Release
-
-Or:
-
-    ::
-
-        git commit -am 'Edit sha256 for v0.4.1'
-        git push <fork-origin> <branch-name>
-        # Create pull request for the master branch
-
-Releasing The Software On Anaconda
-----------------------------------
-
-1. Be sure to have already completed :ref:`Creating A Release On GitHub <github-release>`.
-This triggers the CI/CD workflow that handles Anaconda releases.
+Releasing Anaconda
+------------------
 
+1. Be sure to have already completed :ref:`Releasing On GitHub <github-release>`. This triggers the CI/CD workflow that handles Anaconda releases.
 2. Wait until the CI/CD build is successful. You can view all workflows at `All Workflows <https://github.com/E3SM-Project/zstash/actions>`_.
-
 3. Check the https://anaconda.org/e3sm/zstash page to view the newly updated package.
 
-4. Notify the maintainers of the unified E3SM environment about the new release on the
-`E3SM Confluence site <https://acme-climate.atlassian.net/wiki/spaces/WORKFLOW/pages/129732419/E3SM+Unified+Anaconda+Environment>`_.
-Be sure to only update the ``zstash`` version number in the correct version(s) of
-the E3SM Unified environment. This is almost certainly one of the versions listed under
-“Next versions”. If you are uncertain of which to update, leave a comment on the page
-asking.
+   * Release candidates are assigned the ``e3sm_dev`` label
+   * Production releases are assigned the ``main`` label
 
-Creating a New Version of the Documentation
--------------------------------------------
+4. Notify the maintainers of the unified E3SM environment about the new release on the `E3SM Confluence site <https://acme-climate.atlassian.net/wiki/spaces/WORKFLOW/pages/129732419/E3SM+Unified+Anaconda+Environment>`_.
 
-1. Be sure to have already completed :ref:`Creating A Release On GitHub <github-release>`.
-This triggers the CI/CD workflow that handles publishing documentation versions.
+   * Be sure to only update the ``zstash`` version number in the correct version(s) of the E3SM Unified environment.
+   * This is almost certainly one of the versions listed under “Next versions”. If you are uncertain of which to update, leave a comment on the page asking.
 
-2. Wait until the CI/CD build is successful. You can view all workflows at
-`All Workflows <https://github.com/E3SM-Project/zstash/actions>`_.
+Creating a New Version of the Documentation
+-------------------------------------------
 
-3. Changes will be available on the
-`zstash documentation page <https://e3sm-project.github.io/zstash/>`_.
+1. Be sure to have already completed :ref:`Releasing On GitHub <github-release>`. This triggers the CI/CD workflow that handles publishing documentation versions.
+2. Wait until the CI/CD build is successful. You can view all workflows at `All Workflows <https://github.com/E3SM-Project/zstash/actions>`_.
+3. Changes will be available on the `zstash documentation page <https://e3sm-project.github.io/zstash/>`_.
```

Comment:
You might notice that long bullets are now single line instead of multiple lines.
By sacrificing line length in the `.rst` files, it allows bullet points to be compact and not require line breaks.